### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by ring-ring-ring**
releasenotes als categorie eruit, ik voeg het toe als paragraaf met een eigen header en link naar github

### CODE REVIEW

**Review by TwistMeister — DISMISSED** [review]
LGTM

File: docs/source/modules/modules.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
-=======
+=================
+Releases & modules
+=================
+
+Releases
+********
+
+Find all `releases, releasenotes and artefacts for OpenKAT <https://github.com/minvws/nl-kat-coordination/releases>`_ on Github.
```
Comment (ammar92) [inline docs/source/modules/modules.rst:8]:
```suggestion
Find all `releases, release notes and artifacts for OpenKAT <https://github.com/minvws/nl-kat-coordination/releases>`_ on GitHub.
```

**Review by ammar92 — COMMENTED** [review]
Spelling suggestion and fix for `rstcheck` problem

**Review by ammar92 — APPROVED** [review]

**Comment by underdarknl** [comment]
Seeing that users are asking for the docs and release notes inside their openKAT install, and are asking for a search on those docs, it would make sense to keep the docs in the source tree like @dekkers suggests. 
This would also help us package the correct docs with each running install, and keep them in sync with the reality of what is available on their install.

**Pull request closed without merging**